cpp/tests/community/balanced_edge_test.cpp

```diff
@@ -78,7 +78,7 @@ TEST(balanced_edge, success)
 int main(int argc, char** argv)
 {
   testing::InitGoogleTest(&argc, argv);
-  auto resource = std::make_unique<rmm::mr::cuda_memory_resource>();
```

Comment:
@afender @ChuckHastings I have changed cuda memory resource to cnmem given that its my understanding that we would prefer to test for the pool allocator. Any inputs on this? Should all tests do this?

Reply:
Yes rmmInitialize(nullptr) is fine. We stopped testing all pool/managed configs for all algorithms.

Reply:
This is going to be refactored in 0.15 so better not spend too much time on this.